phpunit.xml.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<phpunit
```

Comment:
Done in https://github.com/Roave/composer-gpg-verify/pull/1/commits/25d3a12d9d67979b145df52c25ea7a59e8d74a6b